Gems/Atom/Feature/Common/Code/Source/ReflectionProbe/ReflectionProbe.cpp

```diff
@@ -120,15 +120,17 @@ namespace AZ
                     m_scene->RemoveRenderPipeline(m_environmentCubeMapPipelineId);
                     m_environmentCubeMapPass = nullptr;
 
-                    // restore exposure
-                    sceneSrg->SetConstant(m_iblExposureConstantIndex, m_previousExposure);
+                    // restore exposures
+                    sceneSrg->SetConstant(m_globalIblExposureConstantIndex, m_previousGlobalIblExposure);
+                    sceneSrg->SetConstant(m_skyBoxExposureConstantIndex, m_previousSkyBoxExposure);
```

Comment:
Should the reflection probe code be reaching into SkyBox code like this? It seems the SkyBoxFeatureProcessor should be involved in the decision about overriding the skybox exposure.

(Sorry, I'm not very familiar with these systems, just coming at this from a fresh perspective).

Reply:
This is a custom render pipeline for baking reflection cubemaps, which is a special case, so there's some overriding that needs to happen here.